arch/arc/Kconfig

```diff
@@ -9,56 +9,61 @@ menu "ARC Options"
 config ARCH
 	default "arc"
 
-choice
-	prompt "ARC core family"
-	default CPU_ARCEM
 
 config CPU_ARCEM
-	bool "ARC EM cores"
-	select CPU_ARCV2
+	bool
```

Comment:
Do `CPU_ARMEM` and `CPU_ARCHS` need to select `ISA_ARCV2` after the next commit?

Reply:
Nope, it' parallel things. IOW, there can be ISA_ARCV2 HS and ISA_ARCV3 HS.